packages/esl-website/views/_includes/landing/newsline.njk

```diff
@@ -32,4 +32,25 @@
       </div>
     </div>
   </div>
+
+  <script>
+    const timeout = 30 * 24 * 60 * 60 * 1000;
+    const now = Date.now();
+
+    document
+      .querySelectorAll('.newsline-content-title[data-date]')
+      .forEach(updateNewsLabel);
+
+    function updateNewsLabel(el) {
+      const timestamp = parseInt(el.dataset.date, 10);
+      const isFresh = timestamp && now - timestamp < timeout;
+
+      if (isFresh) {
+        const badge = document.createElement('sup');
+        badge.className = 'badge badge-small badge-sup badge-success';
+        badge.textContent = 'new';
+        el.appendChild(badge);
+      } 
+    }
+  </script>
```

Comment:
Our site has a valid bundle with full ESL support.
Please move this logic to mixin (custom attribute).
Make sure you name it conventionally with `esl-d-` prexix, suggest `esl-d-new-label` or `esl-d-promo-from`